test/Makefile

```diff
@@ -55,6 +55,8 @@ LIB_DIRS :=
 # Tests
 TEST     :=
 
+MAKEFLAGS += k

```

Comment:
Would it make more sense to call `make -k` rather than trying to edit the arguments from within the makefile?

Reply:
For the CI itself, maybe. But that parameter makes sense when manually called from the console as well, right? With the other change, the tests would stop after the first error which is probably not what you want by default, so setting the flag inside the Makefile seemed more useful.